input.cpp

```diff
@@ -475,7 +478,7 @@ static const int ev2ps2[] =
 	NONE, //191 KEY_F21			
 	NONE, //192 KEY_F22			
 	NONE, //193 KEY_F23			
-	0x5D, //194 U-mlaut on DE mapped to \

```

Comment:
The backslash here will result in the next line being commented out completely which I don't think was the intention?

Reply:
you are right. Fortunately it was the last useful key in the array.